src/main/resources/db.migration/V2__initialize_data_base.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE pinned_schedule
+(
+    student_id BIGINT NOT NULL REFERENCES student,
+    group_id BIGINT NOT NULL REFERENCES "group",
+    PRIMARY KEY (student_id, group_id)
+);
+
```

Comment:
сделай первичным ключом колонку student_id, поскольку у студента может быть только одно закрепленное расписание
```suggestion
CREATE TABLE pinned_schedule
(
    student_id BIGINT PRIMARY KEY REFERENCES student,
    group_id BIGINT NOT NULL REFERENCES "group"
);

```

Reply:
и переименуй скрипт в V2__create_pinned_schedule.sql